awscli/customizations/wizard/ui/keybindings.py

```diff
@@ -47,12 +47,22 @@ def submit_current_answer(event):
         current_prompt = event.app.traverser.get_current_prompt()
         prompt_buffer = get_ui_control_by_buffer_name(
             event.app.layout, current_prompt).buffer
-        event.app.traverser.submit_prompt_answer(prompt_buffer.text)
+        try:
+            event.app.traverser.submit_prompt_answer(prompt_buffer.text)
+            event.app.traverser.exception_handler.clear_if_exception(ValueError)
```

Comment:
Then for this logic, I'm thinking we just expose a `current_error` property and compare the type against that. It seems like that would be a little more general purpose and useful instead of a new clear method. So this logic wold now become something like:
```python
event.app.traverser.submit_prompt_answer(prompt_buffer.text)
if isinstance(event.app.layout.error_bar.current_error, BaseWizardException):
     event.app.layout.error_bar.clear()
```
Then if we find the `BaseWizardException` becomes too general, we can then create a new subclass `BaseWizardValueException` and update other classes appropriately so it reflects any exception that may be raised because of bad prompt input inside of the wizard.